config.yaml.defaults

```diff
@@ -92,10 +92,21 @@ services:
     enabled:
 
     # A list of service names, or '*'
+    # You can use disabled to disable specific services, or '*' to disable them all
     disabled:
       - dask
 
-    # You can use disabled to disable specific services, or '*' to disable them all
+    # Here you can add services that are not part of baselayer, but
+    # come from another github repository. You need to specify URL,
+    # branch+sha or version (tag), and optional . For example:
+    # - my_service:
+    #     url: <url/to/git/repo>
+    #     branch: <branch_name>
+    #     sha: <sha_of_commit>
```

Comment:
`branch` and `sha` could probably be same specified. Like, e.g., for pre-commit you would specify an action to run with `https://github.com/some/action@branch` or `https://github.com/some/action@abc123`.